src/Components/Footer/FooterDownloadAppBanner.tsx

```diff
@@ -16,58 +16,62 @@ const MOBILE_COVER_IMAGE = resized(
 
 export const FooterDownloadAppBanner = () => {
   return (
-    <Box borderBottom="1px solid" borderColor="black10">
-      <GridColumns>
-        <Column
-          span={4}
-          display="flex"
-          flexDirection="column"
-          alignItems="center"
-          justifyContent="center"
-          order={[2, 1]}
-          px={2}
-          py={[6, 2]}
-        >
-          <Text variant="xl" textAlign="center" mb={1}>
-            Get the Artsy app
-          </Text>
+    <GridColumns
+      gridRowGap={1}
+      borderBottom="1px solid"
+      borderColor="black10"
+      pb={1}
```

Comment:
I mean the actual parent context.

Reply:
(Consider how annoying it is to import a component with some kind of external spacing.)

Reply:
Oh no I understand! in this case there's a border around the grid columns to define the footer; the border still represents the edge, but now that there's the caption we need some internal padding so that there's space between the image, caption and border edge. It's not an external margin for layout. 
 
 <img width="1260" alt="Screen Shot 2022-09-27 at 1 00 52 PM" src="https://user-images.githubusercontent.com/236943/192634355-bbedd8ae-c9e7-4374-a556-89235cc1b8c4.png">

Reply:
OH, sorry I missed the border! Proceed!